fix: contribution docs (#148)
datazip-inc/olake-docs: docs/community/contributing.mdx

- [OLake Community Slack](https://olake.io/slack) - Join the community's `#contributing-to-olake` channel for all the information regarding contributions.
- [GitHub Issues](https://github.com/datazip-inc/olake/issues) - Report bugs, request features, and claim tasks; browse issues by label to find work you’d like to tackle and comment to assign yourself.
- [GitHub Projects Board](https://github.com/orgs/datazip-inc/projects/5) - You can also refer to the projects section in GitHub to see who all are working on which issues.

## Orientation

Here's a list of repositories that contain OLake-related packages:

- [datazip-inc/olake](https://github.com/datazip-inc/olake) is the main repository containing the core OLake replication engine and CLI tools
- [datazip-inc/olake-ui](https://github.com/datazip-inc/olake-frontend) contains the web-based user interface for managing OLake jobs and configurations
- [datazip-inc/olake-docs](https://github.com/datazip-inc/olake-docs) contains the documentation website and guides
- [datazip-inc/olake-helm](https://github.com/datazip-inc/olake-helm) contains official Helm charts for deploying OLake on Kubernetes.

> ## Types of Contributions

Reviewer: From here the main parts of the doc should be 
* Reporting Issue
* Contributing to solve an Issue

When submitting your proposal, consider including:

- Requirement — what kind of use case are you trying to solve?
- Proposal — what do you suggest to solve the problem or improve the existing situation?
- Any open questions to address❓

### Fix Bugs

Look through the GitHub issues. Issues tagged with `#bug` are open to whoever wants to implement them.

### Implement Features

Look through the GitHub issues. Issues tagged with `#feature` are open to whoever wants to implement them.

### Complete Tasks

Look through the GitHub issues. Issues tagged with `#task` are open to whoever wants to work on them.

### Reporting Security Issues

Please do not create a public GitHub issue. If you've found a security issue, please email us directly at [hello@olake.io](mailto:hello@olake.io) instead of raising an issue.

### A good start will be to explore:
- [first good issues](https://github.com/datazip-inc/olake/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22)
- [beginner issues](https://github.com/datazip-inc/olake/issues?q=is%3Aissue%20state%3Aopen%20label%3Abeginner)

> See the issues and if its looks interesting enough to you, comment below them and one of our maintainers will get in touch with you.

Reviewer: ```suggestion
See the issues and if its looks interesting enough to you, comment on it and one of our maintainers will reply to you on the same issue.
```
